pkg/registry/search/storage/resourceregistry.go

```diff
@@ -28,8 +28,8 @@ func NewResourceRegistryStorage(scheme *runtime.Scheme, optsGetter generic.RESTO
 		NewFunc:                   func() runtime.Object { return &searchapis.ResourceRegistry{} },
 		NewListFunc:               func() runtime.Object { return &searchapis.ResourceRegistryList{} },
 		PredicateFunc:             searchregistry.MatchResourceRegistry,
-		DefaultQualifiedResource:  searchapis.Resource("resourceRegistries"),
-		SingularQualifiedResource: searchapis.Resource("resourceRegistry"),
+		DefaultQualifiedResource:  searchapis.Resource("resourceregistries"),
+		SingularQualifiedResource: searchapis.Resource("resourceregistry"),
```

Comment:
How about adding some comments to explain it?

Reply:
Nice suggestion!